clippy_lints/src/doc/empty_line_after.rs

```diff
@@ -192,6 +192,16 @@ fn check_gaps(cx: &LateContext<'_>, gaps: &[Gap<'_>]) -> bool {
         return false;
     };
     let empty_lines = || gaps.iter().flat_map(|gap| gap.empty_lines.iter().copied());
+    let contiguous_empty_lines = || {
+        gaps.iter().map(|gap| {
+            let first_line = gap.empty_lines.first().unwrap();
+            let last_line = gap.empty_lines.last().unwrap();
+
+            // The BytePos subtraction here is save, as before an empty line, there must be at least one
```

Comment:
s/save/safe/ ?

Reply:
Thanks. The difference of those two words just won't go into my head. Not even with the mnemonic "saVe is a **V**erb, because it is spelled with **V**"...